CONTRIBUTING.md

```diff
@@ -199,7 +199,10 @@ request, go through this checklist:
    commit history and helps structure the changelog.  
    Please refer to this
    [list of prefixes for Kratos](https://github.com/ory/kratos/blob/master/.github/semantic.yml).
-
+1. Sign-up with CircleCI so that it has access to your repository with the 
```

Comment:
Ahhh, that is the missing piece! @vinckr can you please transfer that into ory/meta?